src/MongoDB.Driver.Core/Core/Operations/AsyncCursor.cs

```diff
@@ -46,7 +46,7 @@ public class AsyncCursor<TDocument> : IAsyncCursor<TDocument>
         // fields
         private readonly int? _batchSize;
         private readonly CollectionNamespace _collectionNamespace;
-        private readonly IChannelSource _channelSource;
```

Comment:
No longer readonly because we're going to set it to null when we no longer need it.